_data/simple-icons.json

```diff
@@ -4412,7 +4415,8 @@
         {
             "title": "Internet Explorer",
             "hex": "0076D6",
-            "source": "https://compass-ssl.microsoft.com/assets/c8/67/c867db4c-f328-45b8-817c-33834c70aae6.svg?n=IE.svg"
+            "source": "https://commons.wikimedia.org/wiki/File:Internet_Explorer_10%2B11_logo.svg",
+            "guidelines": "https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RE1qeRY#:~:text=The%20Internet%20Explorer%2011%20logo%20has%20two%20components%3A%20the%20stylized,based%20on%20the%20Segoe%20typeface."
```

Comment:
This link is scaring me 😅 
Is there a way to find a page that link to this file?

Reply:
I get, but the other link I found doesn't help a lot:
https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RE1qeRY
it's shorter, but it's still a query link to a PDF file.

Reply:
I've updated the guideline to the shorter link. Please check and resolve if you think it's good enough

Reply:
Plz check and resolve.